### PR TITLE
fix: upgrade tempy package

### PR DESCRIPTION
**Pull request by rephus**
Closes: 

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

test-cli test-backend test-frontend

### CODE REVIEW

File: packages/backend/src/services/SavedChartsService/SavedChartService.ts [hunk 1]
```diff
@@ -663,6 +663,7 @@ export class SavedChartService {
             await this.checkCreateScheduledDeliveryAccess(user, chartUuid);
         const scheduler = await this.schedulerModel.createScheduler({
             ...newScheduler,
+            cron: '* * * * *',
```
Comment (ZeRego) [inline packages/backend/src/services/SavedChartsService/SavedChartService.ts:666]: was this change intentional?

**Review by ZeRego — APPROVED** [review]

File: packages/backend/src/services/SavedChartsService/SavedChartService.ts (hunk 1 above)
Comment (rephus) [reply packages/backend/src/services/SavedChartsService/SavedChartService.ts:666]: oh no, sorry , I'll revert

**Comment by rephus** [comment]
this upgrade is still throwing runtime errors

**Comment by rephus** [comment]
image

**Comment by rephus** [comment]
Removed tempy dependency by using `fs.mkdtempSync` . Tested locally with dbt local + github

File: packages/backend/src/projectAdapters/dbtGitProjectAdapter.ts [hunk 2]
```diff
@@ -75,9 +75,7 @@ export class DbtGitProjectAdapter extends DbtLocalCredentialsProjectAdapter {
         dbtVersion,
         useDbtLs,
     }: DbtGitProjectAdapterArgs) {
-        const localRepositoryDir = tempy.directory({
-            prefix: 'git_',
-        });
+        const localRepositoryDir = fs.mkdtempSync('/tmp/git_');
```
Comment (rephus) [inline packages/backend/src/projectAdapters/dbtGitProjectAdapter.ts:78]: by default `mkdtempSync` creates a dir in a local folder (backend) , we need to use absolute paths to use /tmp (like was tempy was using by default)
